ACM-10596: Address custom API domain import (#6078)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/import_ocp.adoc

oc get secret -n openshift-kube-apiserver node-kubeconfigs -ojson | jq '.data["lb-ext.kubeconfig"]' --raw-output | base64 -d > /tmp/kubeconfig.some-other-cluster
----
+

> *Note:* If your cluster's API is accessed through a custom domain, you must first edit this `kubeconfig` by adding your custom certificates under the `certificate-authority-data` field and changing the `server` field to match your custom domain.

Reviewer: ```suggestion
*Note:* If your cluster API is accessed through a custom domain, you must first edit the `kubeconfig` by adding your custom certificates in the `certificate-authority-data` field and by changing the `server` field to match your custom domain.
```

Author: I feel changing `this` to `your` is confusing as in the text I was trying to refer to the kubeconfig mentioned in the documentation directly above this newly added line, not some arbitrary kubeconfig the user happens to have

Author: Fixed other edits

Reviewer: I see, in that case we would just use "the" since we are still in the same step/topic. I'll update the suggestion.

Reviewer: @omertuc Just saw you already committed the changes so I updated it myself. Feel free to merge if everything else looks good. Thanks!
